pkg/services/alerting/notifier.go

```diff
@@ -34,14 +34,15 @@ type notificationService struct {
 	renderService rendering.Service
 }
 
-func (n *notificationService) SendIfNeeded(context *EvalContext) error {
+func (n *notificationService) SendIfNeeded(context *EvalContext) {
```

Comment:
Please add back the return of the error. We're currently doing some other error handling refactorings which handles the error returned from this method.